### PR TITLE
Document billable account closure process

### PR DESCRIPTION
**Pull request by kr8n3r**
What
----

Document billable account closure process

[View rendered](https://github.com/alphagov/paas-team-manual/blob/e4030e7db3a4f06e762d3745ebd7676fc9585f33/source/accounts/close_billable_accounts.html.md)

Who can review
--------------

anyone

### CODE REVIEW

File: source/accounts/close_billable_accounts.html.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1. Tenant confirms that they have migrated from PaaS
+2. Support engineer performs all technical decommissioning tasks.
+ - Suspend org
+ - Remove all users from the org
+ - Delete all backing services
+ - Delete all apps
+ - Spaces can be left alone, because they don’t incur a cost
+3. Support engineer confirms the date when tasks in step 2 were completed, e.g. *“emptied and suspended <date>”* in column E of the Trial and billable accounts tab in [GOV.UK PaaS usage and adoption tracker - 2020-07-06](https://docs.google.com/spreadsheets/d/1bZP7W-5nJxDicJ2lc_eT873zXXjXnHh-qGYJobHUIv8/edit#gid=40753783) 
+4. **For external tenants**: Support engineer replies to the tenant confirming the date of “closure” and informs GDS Business Operations (PMO) of the tenant's closure date by CC-ing them into the ticket (GDS Business Operations or PMO) and marking the ticket as SOLVED. 
+5. **For Internal tenants only**: Ensure that GDS PaaS Tenants billable amounts are tracked in the [tracking tool](https://docs.google.com/spreadsheets/d/1S3yuBHbQAHM_Oj-FOvd5DkZNvBSj5aBvOXrM0gay5_Y/edit#gid=1965040568)
```
Comment (cadmiumcat) [inline source/accounts/close_billable_accounts.html.md:30]: I've just opened this spreadsheet for the first time and I have no idea where anything goes. We need either more detailed instructions or someone who maintains the spreadsheet to do it

Comment (kr8n3r) [reply source/accounts/close_billable_accounts.html.md:30]: can you be specific about which spreadsheet you mean?

Comment (cadmiumcat) [reply source/accounts/close_billable_accounts.html.md:30]: The one in step 5. Sorry, I thought I selected that specific change

Comment (kr8n3r) [reply source/accounts/close_billable_accounts.html.md]: step for internal tenant removed as not applicable (aka no one cares)

File: source/accounts/close_billable_accounts.html.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- has no services or data
+- has no applications
+- is flagged as suspended
+- has no users associated with the Cloud Foundry org
+- incurs no costs to us
+- exists as an empty shell in Cloud Foundry
+
+## Accounts closure process
+
+1. Tenant confirms that they have migrated from PaaS
```
Comment (cadmiumcat) [inline source/accounts/close_billable_accounts.html.md:21]: Do we need to double check that this person is an org owner? who's allowed to close it?

**Review by cadmiumcat — APPROVED** [review]

**Pull request merged**